scenarios/preset/helpers/neofs_cli.py

```diff
@@ -13,18 +13,17 @@ def create_container(endpoint, policy, wallet_file, wallet_config):
         print(f" > Container has not been created:\n{output}")
         return False
     else:
-        try:
-            fst_str = output.split('\n')[0]
-        except Exception:
-            print(f"Got empty output: {output}")
-            return False
-        splitted = fst_str.split(": ")
-        if len(splitted) != 2:
-            raise ValueError(f"no CID was parsed from command output: \t{fst_str}")
-
-        print(f"Created container: {splitted[1]}")
-
-        return splitted[1]
+        # Regular expression to find the container ID, case-insensitive
+        pattern = r"container ID: ([A-Za-z0-9]+)"
```

Comment:
There is a certain length that can be used to strengthen this regex (or it can collide with something else, `dog` matches it as well). It can be `([A-Za-z0-9]{43,44})` (in fact, 0 and some other symbols are incorrect as well, but that doesn't matter much).

Reply:
Added length.